composer.json

```
@@ -14,7 +14,7 @@
     "keywords": ["Laravel", "Boxicons", "Blade UI Kit"],
     "require": {
         "php": "^7.2|^8.0",
-        "blade-ui-kit/blade-icons": "^0.5.0",
+        "blade-ui-kit/blade-icons": "^1.0.0",
         "illuminate/support": "^7.14|^8.0"
```

Blade icons dropped Laravel 7 support with v1.0
> Drop Laravel 7 support
> - https://github.com/blade-ui-kit/blade-icons/blob/1.x/CHANGELOG.md#100-2021-03-21


```suggestion
        "illuminate/support": "^8.0"
```

`orchestra/testbench` v5.0 should also be removed from dev dependencies because of that.